commands/run/index.js

```diff
@@ -58,10 +58,6 @@ class RunCommand extends Command {
         // still exits zero, aka "ok"
         return false;
       }
-
-      this.batchedPackages = this.toposort
```

Comment:
So this is _technically_ a breaking change, since we support passing `--no-sort` to disable the default topological sorting/batching. I understand that it's extremely rare to _want_ stuff to run in alphabetical order, disregarding dependent relationships etc, but it's nonetheless legacy. (It's not even tested very well/at all, for that I apologize)

Is there any way we can preserve `--no-sort` with this new implementation?

Reply:
https://github.com/lerna/lerna/commit/8ccf84466980a3e318f02d51fb583dd18b18bb3f shows what is expected when `--no-sort` is passed.

Reply:
We wouldn't have jobs if it wasn't for legacy ;) Thank you for the tests. Appreciate it. I re-use the logic from `runScriptInPackagesParallel()`, as it's basically the same. Let me know if you think it's ok, or if we should create a better separation between `--no-sort` and `--parallel`.

Reply:
I think this does the trick. Thanks!